cli/packages/gateway/systemd.go

```diff
@@ -15,7 +15,8 @@ Description=Infisical Gateway Service
 After=network.target
 
 [Service]
-Type=simple
+Type=notify
```

Comment:
i don't see anything in the code sending ready to systemd when the service is good to go. looking online, i see you have to send a daemon.SdNotify()?

Reply:
https://github.com/Infisical/infisical/blob/main/cli/packages/gateway/gateway.go#L248